simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogBookDetailFragment.kt

```diff
@@ -576,7 +576,11 @@ class CatalogBookDetailFragment : Fragment(R.layout.book_detail) {
         }
     }
 
-    val isRevocable = this.viewModel.bookCanBeRevoked && this.buildConfig.allowReturns()
+    val isRevocable = (
+      bookStatus is BookStatus.Loaned.LoanedNotDownloaded ||
```

Comment:
@nunommts Would it make sense to move this check into the `bookCanBeRevoked` function? That function is also checking the book status.